Makefile

```diff
@@ -51,6 +51,7 @@ install-sdl2-ncurses:
 test:
 	qlot install
 	.qlot/bin/rove lem-tests.asd
+	.qlot/bin/rove extensions/vi-mode/lem-vi-mode.asd
```

Comment:
There is no way to this be tested in a single rove call? I think that lem-tests system should take care of it, otherwise, we'll have two reports in the ci

Reply:
It seems like the two packages are mutually exclusive; therefore, two separate calls are need in the current setup. I've actually noticed this is the case for most, if not all, the lem extensions: https://github.com/lem-project/lem/blob/main/.github/workflows/test.yml#L46-L47